google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -115,3 +115,30 @@
 """
 
 # TODO: Add a RAW_GROUP_MEMBERS_TABLE.
+
+CREATE_BUCKETS_TABLE = """
+    CREATE TABLE `{0}` (
+        `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+        `project_number` bigint(20) NOT NULL,
+        `bucket_id` varchar(255) DEFAULT NULL,
+        `bucket_name` varchar(255) DEFAULT NULL,
+        `bucket_kind` varchar(255) DEFAULT NULL,
+        `bucket_storage_class` varchar(255) DEFAULT NULL,
+        `bucket_location` varchar(255) DEFAULT NULL,
+        `bucket_create_time` datetime DEFAULT NULL,
+        `bucket_update_time` datetime DEFAULT NULL,
+        `bucket_selflink` varchar(255) DEFAULT NULL,
+        `bucket_lifecycle_raw` json DEFAULT NULL,
+        `raw_bucket` json DEFAULT NULL,
+        PRIMARY KEY (`id`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+"""
+
+CREATE_RAW_BUCKETS_TABLE = """
```

Comment:
Hmmm, I am not sure that a separate table for the raw buckets json data is necessary here, as this info is captured by the "raw_bucket" column in the above "CREATE_BUCKETS_TABLE".  Or else, can you explain what is the consideration here as I might have missed something?

When reading the buckets reference doc, what might need separate tables would be to normalize the repeating list items like:
```
acl[]
cors[]
defaultObjectAcl[]
lifecycle.rule[]
```
https://cloud.google.com/storage/docs/json_api/v1/buckets